model/project_ref.go

```diff
@@ -394,6 +394,14 @@ func (p *ProjectRef) UseRepoSettings() bool {
 	return p.RepoRefId != ""
 }
 
+// GetRepoSettingsId returns the repo settings id if the project is using repo settings, otherwise it returns an empty string
+func (p *ProjectRef) GetRepoSettingsId() string {
```

Comment:
This function isn't really necessary -- all p.UseRepoSettings does is check if p.RepoRefId is empty, i.e. we don't set this if we aren't using repo settings